eng/tools/analyze-deps/package.json

```diff
@@ -14,6 +15,7 @@
     "jju": "^1.4.0",
```

Comment:
This is not maintained anymore, should we use this instead? https://www.npmjs.com/package/json5

Reply:
Yes looks that json5 is more recent and has more weekly downloads